core/app/models/spree/gateway.rb

```diff
@@ -4,7 +4,7 @@ class Gateway < PaymentMethod
 
     delegate :authorize, :purchase, :capture, :void, :credit, to: :provider
 
-    validates :name, :type, presence: true
+    validates :type, presence: true
```

Comment:
Because its already present in the PaymentMethod class from which this one is inherited and then it will run twice